prow/config.yaml

```diff
@@ -319,7 +319,7 @@ presubmits:
     - release-1.5
     - release-1.6
     always_run: true
-    skip_report: true
+    skip_report: false
```

Comment:
Just remove this line? Same for similar lines below.

Reply:
`skip_report` was changed so that you need to explicitly opt in to reporting in https://github.com/kubernetes/test-infra/pull/4563

Reply:
Actually I think I misunderstood that, I read the PR diff and I think we can remove these. :+1:

Reply:
I think it's good to leave it here to be explicit

Reply:
I'm fine with it either way, these flags get flipped around from time to time as well. Let's merge and we can bikeshed more another time :-)